docs/lib/datastore/auth-model.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Models with authorization
+description: Learn more about how DataStore handles Models with finer grain authorization access checks.
+---
+
+DataStore with [sync to cloud enabled](~/lib/datastore/sync.md) has the capability to operate on models with finer grain authorization checks at the API. In GraphQL, this is done with the `@auth` directive defined in the [GraphQL Transformer documentation](~/cli/graphql-transformer/directives.md#auth).
```

Comment:
This sentence is awkward to me.  Here's a suggestion -- we start with a topic sentence that calls out the purpose for reading the rest of the page: "configuring fine grain authorization controls with data store", so that they can decide if they want to continue reading or not.

Maybe something like:

In order to configure fine grain authorization controls with Datastore, we can use the `@auth` directive to specify ownership and permissible operations (create, update, delete) on a specific type.  More information can be found on the `@auth` directive in the GraphQL Transformer documentation.